test/mpi/part/parrived.c

```diff
@@ -50,7 +50,7 @@ static int check_recv_partition(int rp, int iter)
         if (buf[index] != exp_val) {
             if (errs < 10) {
                 fprintf(stderr,
-                        "expected %.1f but received %.1f at buf[%d] (partition %d count %ld off %d), iteration %d\n",
+                        "expected %.1f but received %.1f at buf[%d] (partition %d count %lld off %d), iteration %d\n",
                         exp_val, buf[index], index, rp, rcount, i, iter);
```

Comment:
The compiler will warn if `MPI_Count` is not a `long long`. One work around is to also add a cast to the value -
`,  (long long) rcount, `